.gitignore

```diff
@@ -222,3 +222,4 @@ aux*.pdb
 
 # Screenshot tests
 /artifacts
+yarn.lock
```

Comment:
Huh?

Reply:
update file from installing yarn , TGUI needs it.